express-postgres/server.ts

```diff
@@ -0,0 +1,78 @@
+import express from 'express';
+import { ClientError, errorMiddleware } from './lib/index.js';
+import pg from 'pg';
+
+const app = express();
+const db = new pg.Pool({
+  connectionString: 'postgress://dev:dev@localhost/pagila',
+  ssl: {
+    rejectUnauthorized: false,
+  },
+});
+
+app.get('/api/films', async (req, res, next) => {
+  try {
+    const sql = `select "title"
```

Comment:
In an API you almost always want to just return the entire record, since the client will probably need all the fields. So use '*' instead of just one column.